### PR TITLE
Update 24_Process-Repository-Organization.md

### LINKED ISSUE #212: 2.4 - Process Repository Organization

### PR DESCRIPTION
**Pull request by tobloef**

### CODE REVIEW

File: report/24_Process-Repository-Organization.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
...
+
+![Repository root level](./assets/repo.png)
+ 
+A single Visual Studio solution contains all the files of the project, organized into folders based on the different parts of the application. Some of these folders contain .NET projects, while others contain the configuration for the various DevOps-related services.
+
+This repository structure has worked reasonably well. It is quick and easy to use, when you want to make changes to Grafana you go into the grana folder and so on. But this may be due to the somewhat limited scope of the application. As the repository has grown to include many different services, it has also become harder to get a quick overview of, as the root level now have 28 files and folders and Going forward it may have be valuable to create a further distinction between the source code of the core application and the configuration files for the related services around the application. A poly-repo approach would likely be overkill for this application but would be well suited if a more microservice-style architecture had been used.
```
Comment (Jakobis) [inline report/24_Process-Repository-Organization.md]: Grafana, not grana

**Comment by tobloef** [comment]
Closes #212

File: report/24_Process-Repository-Organization.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
+
+![Repository root level](./assets/repo.png)
+ 
+A single Visual Studio solution contains all the files of the project, organized into folders based on the different parts of the application. Some of these folders contain .NET projects, while others contain the configuration for the various DevOps-related services.
+
+This repository structure has worked reasonably well. It is quick and easy to use, when you want to make changes to Grafana you go into the Grafana folder and so on. But this may be due to the somewhat limited scope of the application. As the repository has grown to include many different services, it has also become harder to get a quick overview of, as the root level now have 28 files and folders and Going forward it may have be valuable to create a further distinction between the source code of the core application and the configuration files for the related services around the application. A poly-repo approach would likely be overkill for this application but would be well suited if a more microservice-style architecture had been used.
```
Comment (Hjaltesorgenfrei) [inline report/24_Process-Repository-Organization.md]: Possibly change the last sentence to:
A poly-repo approach would likely be overkill for this application, but would be well suited if a more services were created by the team.

**Review by Hjaltesorgenfrei — APPROVED** [review]

File: report/24_Process-Repository-Organization.md (hunk 2 above)
Comment (Duckapple) [inline report/24_Process-Repository-Organization.md]: > ... it has also become harder to get a quick overview of
Seems a little much like directly translated Danish, maybe try with "it has also become harder to get a quick overview of **the contents**"

Comment (Jakobis) [reply report/24_Process-Repository-Organization.md]: Maybe change overkill to something else, like "more trouble than it is worth".

**Review by Hjaltesorgenfrei — APPROVED** [review]

**Pull request merged**